module/solutions/helper.js

```diff
@@ -398,16 +420,18 @@ module.exports = class SolutionsHelper {
    * @returns {JSON} solution creation data. 
    */
   
-   static update(solutionId, solutionData, userId) {
+   static update(solutionId, solutionData, userId, checkDate = false) {
```

Comment:
@borkarsaish65 Please add checkDate to function signature

Reply:
done